installKafka.sh

```diff
@@ -23,6 +23,7 @@ EDGEHOST="$2"
 DB_URL="$3"
 DB_NAME="${4}ow_kafka_triggers"
 APIHOST="$5"
+WORKERS="$6"
```

Comment:
Need to update doc for this new parameter. See https://github.com/apache/incubator-openwhisk-package-kafka/blob/master/devGuide.md#install-actions.